scout/dashboard.md

```diff
@@ -79,6 +79,21 @@ Each entry in the list shows the following details:
 - The package type.
 - The number of images that use the package.
 
+### Images using base image
+
+![A screenshot of the Docker Scout view showing images that use a selected base image](./images/dashboard-images-used-by.png)
+
+The **Images using base image** tab shows all images in an organization that use a specific base image.
```

Comment:
**issue:** here are the initial designs for the base image page:

![image](https://github.com/docker/docs/assets/5558761/a012a23d-15e3-4a55-8405-5af3a7d8ecf1)

I noticed we didn't add the `Images` tab on top (even though it's the only one that we will have for now), which is what you can currently see in scout.docker.com. The team is adding this tab, with top priority, so that we don't have to come back and change docs (potentially only the screenshot). We should have it live in the next hour or so, hopefully.

With that in mind, I think this section should probably say `Images tab` instead of `Images using base image tab` or do you believe it's best to keep it as is?

Reply:
@fdiogo True, I think I was using that terminology in my own head to describe what it was showing and forgot to edit own my own internal monologue… And we can wait for the image update certainly.

Reply:
```suggestion
The **Images** tab shows all images in an organization that use a specific base image.
```

Reply:
The change should be live right now! [Example](https://scout.docker.com/reports/org/docker/base-image/host/hub.docker.com/repo/alpine/images)